### ISSUE TITLE
Rails-admin redirects you to the home page after creating a new game

### ISSUE THREAD

**Issue by camdenmoors** [mitre-cyber-academy/ctf-scoreboard#514]
After you create a new game when setting up the board, it redirects you to the home page with a message saying "You are already signed in". It's a pretty small bug but one of the first things someone would experience after setting up the board.

![signedin](https://user-images.githubusercontent.com/66680985/86148776-0e570c00-bac9-11ea-846d-55ea600f3af9.png)

**Comment by rbclark** [comment]
Yeah I noticed this as well, its not ideal so if its an easy fix we should change it. I think the issue is rails-admin is redirecting after the game is created back to the previous page that the user came from (which was the sign-in page)

**Comment by Domur** [comment]
Looks like it is a default behavior by rails admin to redirect to the previous page. After some searching, I wasn't able to find anything that would address this, which leads me to believe it isn't an easy fix.
